test.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: curator.operatorfirst.io/v1alpha1
+kind: Report
+metadata:
+  name: namespace-cpu-request
+spec:
+  reportingStart: '2022-03-01T00:32:00Z'
+  reportingEnd: '2022-12-30T23:59:59Z'
+  namespace: 'default'
```

Comment:
Any idea on why we need a spec.namespace field as a first-class field in the Report API? What's the usage of this field in the existing curator stack?